src/scitacean/client.py

```diff
@@ -829,6 +828,30 @@ def create_attachment_for_dataset(
             model.DownloadAttachment, _strict_validation=False, **uploaded
         )
 
+    def validate_dataset_model(
+        self, dset: Union[model.UploadDerivedDataset, model.UploadRawDataset]
+    ) -> None:
+        """Validate a dataset in SciCat.
+
+        Parameters
+        ----------
+        dset:
+            Model of the dataset to validate.
+
+        Raises
+        ------
+        ValueError
+            If the dataset does not pass validation.
+        """
+        response = self._call_endpoint(
+            cmd="post",
+            url="datasets/isValid",
+            data=dset,
+            operation="validate_dataset_model",
+        )
+        if not response["valid"]:
+            raise ValueError(f"Dataset {dset} did not pass validation in SciCat.")
```

Comment:
Can you change this to a `pydantic.ValidationError` to be in line with the local validation? Or is there a reason to use a different type?